packages/react-components/src/components/ChatMessage/ChatMessageComponent.tsx

```diff
@@ -91,6 +91,12 @@ type ChatMessageComponentProps = {
    */
   onFetchAttachments?: (attachment: FileMetadata) => Promise<void>;
   /* @conditional-compile-remove(teams-inline-images-and-file-sharing) */
+  /**
+   * Optional callback called when an inline image is clicked.
+   * @beta
+   */
+  onInlineImageClicked?: (attachment: FileMetadata, imageName?: string, senderId?: string) => Promise<void>;
```

Comment:
> does contoso need to pass in alt text in this for accessibility?

If Contoso is only using MessageThread as a component, they'll create their own ImageGallery (if they decide to use it) and pass altText there)
If Contoso is using our Chat/CallWithChat composite, I don't think we need to provide that much flexibility to set the alt text. It will be defaulted to `image`.